core/node/libp2p/pnet.go

```diff
@@ -68,3 +72,85 @@ func PNetChecker(repo repo.Repo, ph host.Host, lc fx.Lifecycle) error {
 	})
 	return nil
 }
+
+func DecodeV1PSK(input io.Reader) (*[32]byte, error) {
```

Comment:
TODO: Will need to switch back to depending on go-libp2p-core.

Reply:
@Stebalien Not sure I understand. Thinking about it though, should this return a `PSK`?

Reply:
I mean we've moved this function to go-libp2p-core, we shouldn't define it twice.

Reply:
Right?

Reply:
I removed it from https://github.com/libp2p/go-libp2p-pnet/pull/32 already. Let me update the function signature here now.

Reply:
Np. I was just leaving this as a TODO so we wouldn't forget.